js/ui/codeView.js

```diff
@@ -106,6 +106,47 @@ function _getWindowId(win) {
     return ('window:%d').format(win.get_stable_sequence());
 };
 
+const _ensureHackDataFile = (function () {
+    let keyfile = new GLib.KeyFile();
+    let initialized = false;
+
+    const flatpakPath = '/var/lib/flatpak';
```

Comment:
I'd like it if we could also support (with higher priority) this flaptak in the home directory flatpak installation.

Reply:
That's possible, but we also don't do that for any of the other hack-components...